src/components/Patient/PatientHome.tsx

```diff
@@ -241,14 +241,15 @@ export const PatientHome = (props: any) => {
       !(patientData?.last_consultation?.facility === facilityId)
     );
   };
+  // Commented out as this is no longer used should I completely remove this?
 
-  const isPatientEligibleForNewConsultation = (patientData: PatientModel) => {
-    return patientData.is_active &&
-      (!patientData?.last_consultation ||
-        patientData?.last_consultation?.discharge_date)
-      ? true
-      : false;
-  };
+  // const isPatientEligibleForNewConsultation = (patientData: PatientModel) => {
+  //   return patientData.is_active &&
+  //     (!patientData?.last_consultation ||
+  //       patientData?.last_consultation?.discharge_date)
+  //     ? true
+  //     : false;
+  // };
```

Comment:
You can remove code that is no longer needed instead of commenting out, we always have git commit history if we ever want to refer to removed code 🙂

Reply:
Got It!